### ISSUE TITLE
[Documentation]: reasonable image size test description is unclear

### ISSUE THREAD

**Issue by CsatariGergely** [lfn-cnti/testsuite#1337]
**Describe the bug**
Test description [To check if the CNF has a reasonable image size](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-to-check-if-the-cnf-has-a-reasonable-image-size) does not define what the reasonable image size is. It is not defined if the test is about the image sizes of the Pods of the CNF or about the size of all images of the CNF. 
The [rationale](https://github.com/cncf/cnf-testsuite/blob/main/RATIONALE.md#to-check-if-the-cnf-has-a-reasonable-image-size-reasonable_image_size) contains `5 gig` as the reasonable size.

**Expected behavior**
1 The test case [description ](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-to-check-if-the-cnf-has-a-reasonable-image-size) should contain the definition of reasonable image size and the definition should be more precise, than `gig`.
2 The test case description should explain if this image size limit is for the images separately or for the sum of all images in a CNF.
3 The rationale should explain what is the operational problem with large image sizes instead of pointing to "monolithic application".

**Comment by kosstennbl** [comment]
Agree, current test documentation for "reasonable_image_size" lacks detail and explanations.

**Comment by Smitholi67** [comment]
documentation has now been updated
